application/ingestr/util.sh

```
@@ -0,0 +1,75 @@
+#/usr/bin/env bash
```

Shellcheck (0.10.0)

[error] Use #!, not just #, for the shebang.

(SC1113)